packages/airnode-utilities/README.md

```diff
@@ -0,0 +1,3 @@
+# `@api3/airnode-utilities`
+
+airnode-utilities provides common utilities for Airnode.
```

Comment:
haha I guess this is a WIP 😄

Reply:
I wasn't sure if a new monorepo package was a supported approach :stuck_out_tongue:

Reply:
Nit: Prefix it with ` >` so it's a bit nicer (but we are not consistent in Airnode even now, so...)|

Also create a JIRA ticket to enhance this a bit. Also, create issues for the things that should belong in here (I forgot now, but I remember mentioning them at the call yesteday)

Reply:
Nit: Prefix it with ` >` so it's a bit nicer (but we are not consistent in Airnode even now, so...)|

Also create a JIRA ticket to enhance this a bit. Also, create issues for the things that should belong in here (I forgot now, but I remember mentioning them at the call yesteday)